### PR TITLE
Updates to Dashboard & Page Views

### PR DESCRIPTION
**Pull request by LadyKerr**
# Description

Describe change or new feature here.

## Checklist

Remove any items which are not applicable.

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works AND the tests pass

### CODE REVIEW

File: src/components/navbar/NavbarUser.js [hunk 1]
```diff
@@ -25,16 +21,14 @@ import { navbarLinksStyle } from "./navbarStyle";
 
 const NavbarUser = props => {
   const { classes, user } = props;
-  const [anchorEl, setAnchorEl] = useState(null);
+  const [anchorEl, setAnchorEl] = useState(false);
```
Comment (micahjones13) [inline src/components/navbar/NavbarUser.js]: Lol, nice catch

File: src/App.js [hunk 2]
```diff
@@ -152,14 +152,15 @@ class App extends React.Component {
...
-            <Route path="/update_profile" component={EditProfileView} />
+            <Route path='/profile/:uid' component={NewUserProfile} />
+            <Route path='/messaging' component={Messaging} />
+            <Route path='/applicationstatus' component={AwaitingApproval} />
+            <Route path='/mentor-table' component={MentorTable} />
+            <Route path='/teacher-table' component={TeacherTable} />
+            <Route path='/mentor_dashboard' component={DashboardMentor} />
+            <Route path='/teacher_dashboard' component={DashboardTeacher} />
+            <Route path='/update_profile' component={EditProfileView} />
+            <Route path='/user-approval' component={UserApproval} />
```
Comment (Full-lifey) [inline src/App.js]: Added user approval component for Admins

File: src/components/dashboard/admin-dashboard/AdminDashboard.js [hunk 3]
```diff
@@ -107,109 +83,20 @@ const useStyles = makeStyles(theme => ({
 }));
 
 const AdminDashboard = props => {
-  console.log(props);
```
Comment (Full-lifey) [inline src/components/dashboard/admin-dashboard/AdminDashboard.js]: Moved all of this to the UserApproval Component

File: src/components/dashboard/admin-dashboard/AdminDashboard.js [hunk 4]
```diff
@@ -107,109 +83,20 @@ const useStyles = makeStyles(theme => ({
...
-          <IconButton onClick={handleDrawerClose}>
-            <ChevronLeftIcon />
-          </IconButton>
-        </div>
-        <Divider />
-        <List>{mainListItems}</List>
-        <Divider />
-        <List>{secondaryListItems}</List>
-      </Drawer>
+      <SideBar />
```
Comment (Full-lifey) [inline src/components/dashboard/admin-dashboard/AdminDashboard.js]: Started using the same sidebar for all users.

File: src/components/dashboard/admin-dashboard/UserApproval.js [hunk 5]
```diff
@@ -0,0 +1,124 @@
+import React, { useState, useEffect } from "react";
```
Comment (Full-lifey) [inline src/components/dashboard/admin-dashboard/UserApproval.js]: Moved into its own component

File: src/components/navbar/Navbar.js [hunk 6]
```diff
@@ -110,33 +110,37 @@ const Navbar = props => {
...
+        <Hidden smDown implementation='css'>
           {config ? rightLinks : ""}
         </Hidden>
-        <Hidden smDown implementation="css">
-          {auth.isLoaded && auth.isEmpty ? "" : <NavbarUser user={auth} />}
+        <Hidden smDown implementation='css'>
+          {auth.isLoaded && auth.isEmpty ? (
+            ""
+          ) : (
+            <NavbarUser user={auth} history={props.history} />
```
Comment (Full-lifey) [inline src/components/navbar/Navbar.js]: added history props so it would not need a withRouter export

File: src/components/navbar/navConfig.js [hunk 7]
```diff
@@ -26,89 +26,89 @@ const navConfig = {
...
     brand: "Future Hope School in the Sky",
     links: [
       {
-        href: '/mentors',
-        text: 'View Mentors', 
+        href: "/mentors",
+        text: "View Mentors",
         auth: true
       },
+      // {
```
Comment (Full-lifey) [inline src/components/navbar/navConfig.js]: I really got sick of looking at this.

File: src/components/shared/components/Sidebar/SideBar.js [hunk 8]
```diff
@@ -0,0 +1,114 @@
+import React from "react";
```
Comment (Full-lifey) [inline src/components/shared/components/Sidebar/SideBar.js]: Reusable sidebar nav!!!!

File: src/components/shared/components/Sidebar/SideBar.scss [hunk 9]
```diff
@@ -0,0 +1,6 @@
+@media (min-width: 900px) {
```
Comment (Full-lifey) [inline src/components/shared/components/Sidebar/SideBar.scss]: Added so the content doesn't fall underneath the sticky navbar. More CSS will be needed

**Pull request merged**